### ISSUE TITLE
Min feels sluggish

### ISSUE THREAD

**Issue by cayenne3** [minbrowser/min#1527]
Hello,

Love the browser! Thank you for all your hard work.

I recently upgraded min from 1.16.x to the latest version (1.18.1) and noticed it feels sluggish. I don't recall if 1.16.x had transitions but in 1.18.1 it feels like min hangs sometimes trying to fade in/out of popups and between tabs. Is there a command argument I can start min with that disables transitions or other features slowing it down?

**Comment by PalmerAL** [comment]
Hmm, sorry that's happening. I think the transitions are all the same as in the older version, although I'm not sure exactly which transitions you're referring to - when you say "fade in/out of popups", which part of the UI are you referring to?

Also, which OS / version?

**Comment by PalmerAL** [comment]
I'm going to close this for now, since we're going to need more detailed information to figure this out.

We recently discovered an issue where disabling the "collect usage statistics" option could cause high background CPU usage (#1559), which could explain what you were seeing. That's fixed in v1.19.1, so you could try installing that and see if it resolves your issue.

**Comment by cayenne3** [comment]
By transitions I was just referring to opening or switching between tabs (the time between clicking on the tab to the time the tab contents appear). Just switching back and forth between tabs would have a longer delay than in 1.16.1. Also, general website JavaScript transitions (such as the jQuery fadeIn and fadeOut functions) felt really choppy. This was tested on Devuan Beowulf 3.0 running on a Raspberry Pi 4.

I tested 1.19.2 and can confirm the issue is resolved. Thank you! :)

I went back to test 1.18.1 and the issue persists in that version, although turning on or off the "Send usage statistics" option in Preferences did not change the performance.
